scanpy/external/tl/_execute_r_script.py

```diff
@@ -0,0 +1,121 @@
+from typing import Optional, Tuple
+from pathlib import Path
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import subprocess
+import os
+
+HERE = Path(__file__).parent
+
+
+def execute_r_script(
+    rscript_path: str = None,
+    r_filename: str = None,
+    arguments: Optional[list] = None,
+    verbosity: Optional[bool] = False,
+):
+    """\
+
+    Helper function that runs given R file in terminal using python subprocess module with the Rscript command
+
+    Parameters
+    ----------
+    rscript_path
+        Path to local directory of Rscript.exe
+    r_filename
+        Filename of R script to run
+    arguments
+        optional list of arguments to add to command
```

Comment:
Are there defaults/examples of these arguments? Coming in from an outside perspective, not sure what these are/could possibly be